packages/pluggableWidgets/datagrid-date-filter-web/typings/StringPrototypeReplaceAll.d.ts

```diff
@@ -0,0 +1,6 @@
+declare module "string.prototype.replaceall" {
+    export default {
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        shim: () => {}
```

Comment:
Can we not just add a typing of something like `() => void` instead of an actual function?

Reply:
The problem is the default :(